optimade/validator/validator.py

```diff
@@ -976,23 +981,57 @@ def _test_versions_endpoint_content(
                     f"Version numbers reported by `/{CONF.versions_endpoint}` must be integers specifying the major version, not {text_content}."
                 )
 
-        content_type = headers.get("content-type")
-        if content_type is not None:
-            content_type = [_.replace(" ", "") for _ in content_type.split(";")]
-        if not content_type or content_type[0].strip() != "text/csv":
+        content_type = response.headers.get("content-type")
+        if not content_type:
             raise ResponseError(
-                f"Incorrect content-type header {content_type} instead of 'text/csv'"
+                "Missing 'Content-Type' in response header from `/versions`."
             )
 
-        for type_parameter in content_type:
-            if type_parameter == "header=present":
-                break
-        else:
+        content_type = [_.replace(" ", "") for _ in content_type.split(";")]
+
+        self._test_versions_headers(
+            content_type, ("text/csv", "text/plain"), optional=True
+        )
+        self._test_versions_headers(content_type, "header=present", optional=True)
+
+        return response, "`/versions` endpoint responded correctly."
+
+    @test_case
+    def _test_versions_headers(
+        self,
+        content_type: Dict[str, Any],
+        expected_parameter: Union[str, List[str]],
+    ) -> Tuple[Dict[str, Any], str]:
+        """Tests that the `Content-Type` field of the `/versions` header contains
+        the passed parameter.
+
+        Arguments:
+            content_type: The 'Content-Type' field from the response of the `/versions` endpoint.
+            expected_paramter: A substring or list of substrings that are expected in
+                the Content-Type of the response. If multiple strings are passed, they will
+                be treated as possible alternatives to one another.
+
+        Raises:
+            ResponseError: If the expected 'Content-Type' parameter is missing.
+
+        Returns:
+            The HTTP response headers and a summary string.
+
+        """
+
+        if isinstance(expected_parameter, str):
+            expected_parameter = [expected_parameter]
+
+        if not any(param in content_type for param in expected_parameter):
             raise ResponseError(
-                f"Missing 'header=present' parameter in content-type {content_type}"
+                f"Incorrect 'Content-Type' header {';'.join(content_type)!r}.\n"
+                f"Missing expected parameter(s): {expected_parameter!r}"
```

Comment:
Minor change, but I think it makes for more clarity, should this fail.
Could even at an "at least"?
```suggestion
                f"Missing (one of) the expected parameter(s): {expected_parameter!r}"
```